_docs/developer/development_instructions/localization.md

```diff
@@ -5,4 +5,49 @@ redirect_from:
   - /developer/migrations
 ---
 
-to be written
\ No newline at end of file
+To contribute to the localization effort for Submitty, follow these steps to update twig templated files and provide translations for various text elements on the website. By following these steps, you'll be ensuring that the platform is accessible to users in various languages and regions.
+
+* **Updating Twig Templated Files**
+
+Begin by making a pull request in the Submitty repository. The purpose of this pull request is to update the twig template files.
```

Comment:
Indent this text to match the bullet point of "Updating Twig Templated Files" so that the final Markdown looks more cohesive.